README.md

```diff
@@ -14,16 +14,15 @@
 
     curl -s https://laravel.build/example-app | bash
 
-2. Clone the repository
-3. Get into your project directory (`cd api`)
-4. Build the docker containers using `./vendor/bin/sail build`
-5. Run the containers using `./vendor/bin/sail up -d`
-6. Access the PHP container using `./vendor/bin/sail shell`
-7. Run `composer install` to install of the composer dependencies.
-8. Rename the docker example `.env` file using `cp .env.example .env`
-9. Run `php artisan key:generate` to generate an application key (`APP_KEY`)
-10. Run `php artisan migrate` to run all migrations and database seeders
-11. Access the site using `learnhub.test:8000` in your browser
+1. `git clone git@github.com:learnhubmkd/api.git` - clone the repository
+1. `cd api` Get into your project directory 
+1. `composer install` - if you don't have it, please [install](https://getcomposer.org/)
```

Comment:
You don't have to install PHP or composer on the operating system, this command can be added here:

https://laravel.com/docs/10.x/sail#installing-composer-dependencies-for-existing-projects

Reply:
yes, sail is a wrapper around docker, docker-compos up will do the same.. but this was for "junior" approach